package.json

```
@@ -25,6 +25,7 @@
   "files": [
     "src",
     "dist/src",
+    "dist/index.min.js",
```

This is what we do elsewhere in the libp2p stack, it means you include the [typedoc-urls](https://unpkg.com/browse/libp2p@0.43.2/dist/typedoc-urls.json) index as well.

```suggestion
    "dist",
```